### ISSUE TITLE
pytorch version dependency issue

### ISSUE THREAD

**Issue by min-xu-ai** [Lightning-AI/lightning#15183]
### Bug description

Please see issue here. 

https://github.com/pypa/pip/issues/11524#issuecomment-1283381523

Is lightning depending on `1.9.*` somewhere as mentioned in the issue there?

### How to reproduce the bug

```python
See https://github.com/pypa/pip/issues/11524#issuecomment-1283381523
```

### Error messages and logs

see https://github.com/pypa/pip/issues/11524#issuecomment-1283381523

### Environment

see https://github.com/pypa/pip/issues/11524#issuecomment-1283381523

cc @Borda

**Comment by awaelchli** [comment]
Does this fix work for you @min-xu-ai? #15184
I'm not sure I understand fully what needs to be done, but let's try this.

**Comment by carmocca** [comment]
Closing. Resolved in https://github.com/pypa/pip/issues/11524
